waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -1098,7 +1116,7 @@ proc readPersistentRlnCredentials*(path: string) : RlnMembershipCredentials {.ra
   debug "Deserialized Rln credentials", rlnCredentials=deserializedRlnCredentials
   result = deserializedRlnCredentials
 
-proc mountRlnRelay*(node: WakuNode, conf: WakuNodeConf|Chat2Conf, spamHandler: Option[SpamHandler] = none(SpamHandler), registrationHandler: Option[RegistrationHandler] = none(RegistrationHandler)): RlnRelayResult[bool] {.raises: [Defect, ValueError, IOError, CatchableError, Exception].} =
+proc mount(node: WakuNode, conf: WakuNodeConf|Chat2Conf, spamHandler: Option[SpamHandler] = none(SpamHandler), registrationHandler: Option[RegistrationHandler] = none(RegistrationHandler)): RlnRelayResult[bool] {.raises: [Defect, ValueError, IOError, CatchableError, Exception].} =
```

Comment:
Could we perhaps make this proc def multiline with indents (and the same for `mountRlnRelay` further down)?

Reply:
Addressed in [cec453f](https://github.com/status-im/nwaku/pull/1185/commits/cec453f9df432ed6bd898a31cdbcbf9c20c8861c)